library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -194,8 +189,11 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
         isPaddingMiddle = a.getBoolean(R.styleable.PagerSlidingTabStrip_pstsPaddingMiddle, isPaddingMiddle);
         tabTypefaceStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextStyle, Typeface.BOLD);
         tabTypefaceSelectedStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextSelectedStyle, Typeface.BOLD);
-        tabTextAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextAlpha, HALF_TRANSP);
-        tabTextSelectedAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextSelectedAlpha, OPAQUE);
+        tabTextColorSelected = a.getColorStateList(R.styleable.PagerSlidingTabStrip_pstsTextColorSelected);
+        if (tabTextColorSelected == null) {
+            tabTextColorSelected = tabTextColor;
+        }
+
```

Comment:
Thank you very much for your work on this PR @yolapop @ataulm  . I really appreciate it. I am just not sure about the selection of default colours approach to achieve the behaviour we had until now, the default material one, with a tab text color selected opaque and the other with transparency. To do that with this implementation we need to actually define `textColor` and `textColorSelected`. So I have thought about this:

```java
     if (tabTextColorSelected == null) {
            tabTextColorSelected = getColorStateList(textPrimaryColor);
    }
```
And when we define the default tabTextColor in the like [159](http://goo.gl/KC9HzG), we can give transparency to the `textPrimaryColor` if not `textColor` was defined, like that:

```java
 if (colorStateList != null) {
            tabTextColor = colorStateList;
 } else {
            String transparency = "77";
            String textPrimaryColorTransp = "#"+transparency+Integer.toHexString(textPrimaryColor).substring(2);
            tabTextColor = getColorStateList(Color.parseColor(textPrimaryColorTransp));
 }
```

This way we have a default behaviour as we had before and only if you wish to define different tab text colors than `textPrimaryColor` you will need to define the attributes `textColor` and `textColorSelected` .

What do you think? I will make the changes